QUEST_JOBSTEP.tsv

```diff
@@ -2822,29 +2822,29 @@ QUEST_JOBSTEP_20150714_002821	{memo X}The Peltasta Master handed over a mission
-QUEST_JOBSTEP_20150714_002830	Psychokino told you that he will help you to spurt your real abilities, but he wants the Essence of Demons as a compensation. Defeat Minotaur and Chapparitions obtain the Essence of Demons.
-QUEST_JOBSTEP_20150714_002831	The Sapper Master asked you to help his experiment. First, collect the Burnable Juice of Tontulias from Zachariel Crossroads.
-QUEST_JOBSTEP_20150714_002832	The Pyromancer Master requested to you something in return for teaching high level magic. Put the Pot of Sun on Ramstis Ridge.
-QUEST_JOBSTEP_20150714_002833	Collect hard barks of Varvs and rigid horns of Zinutekas which are the ingredients Cryomancer wants from the 1st floor of Royal Mausoleum.
+QUEST_JOBSTEP_20150714_002828	Get Glizardon Horns from Tenet Chapel Basement 1F, Galok Feathers from 1F, {nl}and Inoum Fragments from 2F that the Wizard Master wants.
+QUEST_JOBSTEP_20150714_002829	The Cleric Master asked you to help curing the patients. Obtain the Holy Water from the Priest Master and the Krivis Master and cure the patients at Miners' Village.
+QUEST_JOBSTEP_20150714_002830	The Psychokino Master told you that he will help you to spurt your real abilities, but he wants demonic essences as a compensation. Defeat Minotaur and Chapparitions and obtain demonic essences.
+QUEST_JOBSTEP_20150714_002831	The Sapper Master asked you to help his experiment. First, collect the Burnable Juice from Tontulias from Zachariel Crossroads.
+QUEST_JOBSTEP_20150714_002832	The Pyromancer Master requested to you something in return for teaching high level magic. Put the Pot of the Sun on Ramstis Ridge.
+QUEST_JOBSTEP_20150714_002833	Collect Varb Hard Bark and Zinutekases Strong Horn which are the ingredients Cryomancer wants from the 1st floor of Royal Mausoleum.
 QUEST_JOBSTEP_20150714_002834	The Sadhu Master wants to know whether you are ready to accept the training. Duel against the Sadhu Master and win.
 QUEST_JOBSTEP_20150714_002835	Talk to the Wugushi Master in Gateway of Great King
 QUEST_JOBSTEP_20150714_002836	The Wugushi Master says that even if cowardly, the last one standing is the true winner of the combat. Have a duel with the Wugushi Master.
-QUEST_JOBSTEP_20150714_002837	Enhance Shield type item up 4 levels.
-QUEST_JOBSTEP_20150714_002838	The Rodelero Master asked you to being shield that is enhanced 4 levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition.
-QUEST_JOBSTEP_20150714_002839	Successfully enhanced shield type item to 4 levels. Report to the Rodelero Master.
+QUEST_JOBSTEP_20150714_002837	Enhance Shield type item up four levels.
+QUEST_JOBSTEP_20150714_002838	The Rodelero Master asked you to being shield that is enhanced four levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition.
```

Comment:
The Rodelero Master asked you to being shield that is enhanced four levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition. 

to

The Rodelero Master asked you to bring your shield with an enhance level of 4. Enhance a shield-type item and bring it to the Rodelero Master to earn his recognition.